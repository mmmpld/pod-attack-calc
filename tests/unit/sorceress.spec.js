import { describe, it } from 'vitest';
import breakpoints from './breakpoints.js'
const characterId = 6;

describe('Sorceress default values', () => {
    it('standard attack', async () => {
        await breakpoints('14', '1.78', { charactersSelected: characterId });
    })

    it('multishot upped doomslinger', async () => {
        await breakpoints('11', '2.27', { charactersSelected: characterId, weaponsPrimarySelected: 46, iasOffWeapon: 10 });
    })
})