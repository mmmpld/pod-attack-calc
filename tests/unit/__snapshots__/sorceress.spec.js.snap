// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sorceress default values multishot upped doomslinger 1`] = `
<tbody>
  <tr class="">
    <th>0</th>
    <td>12</td>
    <td>2.08</td>
  </tr>
  <tr class="highlight-current-row">
    <th>8</th>
    <td>11</td>
    <td>2.27</td>
  </tr>
</tbody>
`;

exports[`Sorceress default values standard attack 1`] = `
<tbody>
  <tr class="highlight-current-row">
    <th>0</th>
    <td>14</td>
    <td>1.78</td>
  </tr>
  <tr class="">
    <th>9</th>
    <td>13</td>
    <td>1.92</td>
  </tr>
  <tr class="">
    <th>19</th>
    <td>12</td>
    <td>2.08</td>
  </tr>
  <tr class="">
    <th>32</th>
    <td>11</td>
    <td>2.27</td>
  </tr>
  <tr class="">
    <th>54</th>
    <td>10</td>
    <td>2.5</td>
  </tr>
  <tr class="">
    <th>86</th>
    <td>9</td>
    <td>2.77</td>
  </tr>
  <tr class="">
    <th>152</th>
    <td>8</td>
    <td>3.12</td>
  </tr>
</tbody>
`;
